CHANGELOG.md

```diff
@@ -35,6 +50,10 @@ All notable changes to the Wazuh app project will be documented in this file.
 - Migrate the timeFilter, metaFields, maxBuckets health checks inside the pattern check. [#5384](https://github.com/wazuh/wazuh-kibana-app/pull/5384)
 - Changed the query to search for an agent in `management/configuration`. [#5485](https://github.com/wazuh/wazuh-kibana-app/pull/5485)
 - Changed the search bar in management/log to the one used in the rest of the app. [#5476](https://github.com/wazuh/wazuh-kibana-app/pull/5476)
+  <<<<<<< HEAD
+  =======
```

Comment:
**nitpick**: I think the conflict was not correctly solved.

Reply:
Done